medusa/updater/update_manager.py

```diff
@@ -0,0 +1,40 @@
+# coding=utf-8
+
+from __future__ import unicode_literals
+
+import logging
+from distutils.version import LooseVersion
+
+from medusa import app
+from medusa.logger.adapters.style import BraceAdapter
+
+from six import text_type
+
+
+log = BraceAdapter(logging.getLogger(__name__))
+log.logger.addHandler(logging.NullHandler())
+
+
+class UpdateManager(object):
+    def __init__(self):
+        """Update manager initialization."""
+        # Initialize the app.RUNS_IN_DOCKER variable
+        # self.runs_in_docker()
+
+    @staticmethod
+    def get_github_org():
+        return app.GIT_ORG
+
+    @staticmethod
+    def get_github_repo():
+        return app.GIT_REPO
+
+    @staticmethod
+    def get_update_url():
+        return app.WEB_ROOT + '/home/update/?pid=' + text_type(app.PID)
+
+    def is_latest_version(self):
+        """Compare the current installed version with the remote version."""
+        if LooseVersion(self.newest_version) > LooseVersion(self.current_version):
```

Comment:
newest_version and current_version don't refer to anything in this class. Imo it's cleanest to add them in UpdateManager, as methods and raise exception when not implemented subclassed. Like:

```
    def newest_version(self):
        """Docstring"""
        raise NotImplmentedError

    def current_version(self):
        raise NotImplmentedError
```